mantra.md

```diff
@@ -22,10 +22,43 @@
 8. ACK Triangle
 ===============
 
+Contrary to belief of many new-age engineering shops; the utmost elegant
+products must be built in a environment constricted by bureaucratic hurdles.
+Not only does this does it prove that the product is fit enough for survival
+for years to come, but also serves as ego re-enforcement of the involved
+fiefdoms.
+
+While it is true that, the more fiefdoms a change must clear before it is
+declared viable, the better. There are diminishing returns. New-agers
+strive for one, but its obvious that with only one there cannot be conflict.
+Two is ofter sufficient, but our research shows; three is optimal. This mostly
+comes down to optimization problem: maximum argument with minimal likely hood
+to enact change.
+
+The most naive observer might assume that all things equal a party trying to
+enact change has equal footing with one trying to prevent it. But this is not
+the case. Preventing change is more potent; as it can always lean back on the
+certainty of the status quo. Any theory of enactor can be detracted by its
+uncertainty (and thus risk). Quantitatively, one force of detractment slightly
+exceeds two of enactment. And so with three fiefdoms one is guaranteed that
+change can be stalemated with only one detractment.
+
+The three fiefdoms are broken down by the following three roles:
+ * Those who Control invention of the un-impelentable
+ * Those with Knowledge to implement the un-inventable
+ * Those with Access to unimplement the invented
```

Comment:
Slightly better wording I think, and also sorted in the right order:
 * Those who possess the Access to technical resources necessary to make the change
 * Those who possess the Control to sign off on the proposed change
 * Those who possess the Knowledge of how to actually use the relevant resources to implement the change